src/main/java/ca/corefacility/bioinformatics/irida/ria/config/BreadCrumbInterceptor.java

```diff
@@ -165,17 +148,18 @@ private boolean hasGoodPath(String path) {
 	}
 
 	/**
-	 * Check to ensure that the {@link ModelAndView} exists and is not in a redirect
+	 * Check to ensure that the {@link ModelAndView} exists and is not in a
+	 * redirect
 	 *
-	 * @param modelAndView {@link ModelAndView}
+	 * @param modelAndView
+	 *            {@link ModelAndView}
 	 * @return true if the {@link ModelAndView} is good for breadcrumbs
 	 */
 	private boolean hasGoodModelAndView(ModelAndView modelAndView) {
-		return modelAndView != null && !modelAndView.getViewName()
-				.contains("redirect:");
+		return modelAndView != null && !modelAndView.getViewName().contains("redirect:");
 	}
 
-	private static class BreadCrumb {
```

Comment:
Removed `private` from sub class decalaration which makes it package level accessible (i.e. any files in the same folder as this file). This was to remove `unused` warnings. I could have supressed those directly on the methods but I thought this was ok.